### PR TITLE
[Part 2] Global settings updates to tutorials

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 1]
```diff
@@ -228,14 +234,14 @@ MacOS users can save a command shortcut that will launch the RDP flow.
...
- echo "`which cloudflared` access rdp --hostname monday.example.com --url localhost:2244 &" > $var &&
+ echo "`which cloudflared` access rdp --hostname monday.example.com --url localhost:3389 &" > $var &&
  chmod +x $var
  ```
 
 1. Check that everything is successful by running the following command:
 
  ```sh
- $ lsof -nP -iTCP:2244 | grep LISTEN
+ $ lsof -nP -iTCP:3389 | grep LISTEN
```
Comment (abracchi-tw) [inline products/cloudflare-one/src/content/tutorials/rdp.md:244]: @TownLake I changed it here too, let me know if I shouldn't have

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 2]
```diff
@@ -24,51 +24,58 @@ You can connect to machines over RDP using Cloudflare's Zero Trust platform.
...
+  ![Configure](../static/zero-trust-security/ssh/configure-app.png)
 
-Finally, click **Save** to save the policy. You can return to edit the policy to make changes to who should be allowed or to choose what authentication providers can be used.
+1. Next, create rules that control who can reach the application.
+
+  ![Add Rules](../static/zero-trust-security/ssh/app-rules.png)
+
+1. Finally, click **Save** to save the policy.
+
+You can return to edit the policy to make changes to who should be allowed or to choose what authentication providers can be used.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md]: I'm not entirely sure of the meaning of this sentence. Is it missing something? Maybe if we break it into smaller sentences?

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 3]
```diff
@@ -24,51 +24,58 @@ You can connect to machines over RDP using Cloudflare's Zero Trust platform.
...
 
-```bash
-C:\Cloudflared\bin\cloudflared.exe service install
-```
+  ```bash
+  C:\Cloudflared\bin\cloudflared.exe service install
+  ```
 
-By default, the agent will run as a Local Account service and will look for the configuration and certificate file in the systemprofile.
+  By default, the agent will run as a Local Account service and will look for the configuration and certificate file in the systemprofile.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md:71]: Should `systemprofile` be between ``?

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 4]
```diff
@@ -24,51 +24,58 @@ You can connect to machines over RDP using Cloudflare's Zero Trust platform.
...
 
-Run the following command to create a new directory within systemprofile, replacing the `System32` value if needed to match your architecture.
+1. Run the following command to create a new directory within systemprofile, replacing the `System32` value if needed to match your architecture.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md:73]: If you decide to place `systemprofile` between ``update here as well

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 5]
```diff
@@ -176,22 +182,22 @@ We recommend that you run `cloudflared` [as a service](/connections/connect-apps
 
 ## Connect from a client machine
 
-### 1. Install cloudflared On The Client Machine
+### Install cloudflared On The Client Machine
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md]: Should this be sentence case?

File: products/cloudflare-one/src/content/tutorials/rdp.md [hunk 6]
```diff
@@ -176,22 +182,22 @@ We recommend that you run `cloudflared` [as a service](/connections/connect-apps
...
 
-Follow steps 1 through 3 above to download and install cloudflared on the client desktop that will connect to the remote desktop.
-cloudflared will need to be installed on each user device that will connect.
+Follow steps 1 through 3 above to download and install `cloudflared` on the client desktop that will connect to the remote desktop.
+`cloudflared` will need to be installed on each user device that will connect.
 
-### 2. Connect To The Remote Desktop
+### Connect To The Remote Desktop
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/tutorials/rdp.md]: Sentence case?

**Review by marciocloudflare — APPROVED** [review]

File: products/cloudflare-one/src/content/tutorials/rdp.md (hunk 2 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/tutorials/rdp.md]: you're right. it makes sense in my head but it could definitely be more readable

**Pull request merged**